apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -543,32 +536,6 @@ function handleCloseEditor( calypsoPort ) {
 	} );
 }
 
-// The close button is generally overridden using the <MainDashboardButton> slot API
-// which was introduced in Gutenberg 8.2. In older editors we still need to override
-// the click handler so that the link will open in the parent frame instead of the
-// iframe. We try not to add the click handler if <MainDashboardButton> has been used.
-function handleCloseInLegacyEditors( handleClose ) {
```

Comment:
Seems ripe for removal — Gutenberg 8.2 is already in WordPress core at this point.